documents/policies/Competition Requirements.md

```diff
@@ -0,0 +1,150 @@
+# WCA Competition Requirements Policy
+<div class="version">
+### Version 1.0
+</div>
+
+## Purpose
+The purpose of this policy is to define the requirements and processes that a competition must meet and follow in order to be recognized as a WCA Competition.
+
+## Policy
+### WCA Competitions
```

Comment:
The second paragraph is named "Announcement Criteria" and contains criteria. This paragraph is named "WCA Competitions" but obviously doesn't contain competitions ;). Long story short: This should be named "WCA Competition criteria" or "WCA Competition requirements".